### PR TITLE
Changes filter test, fixing error on add-filter-from-agent-events.fea…

### PR DESCRIPTION
**Pull request by Mayons95**
This PR solves the problem with the filters from agent/events page.
Also includes a refactor by implementing backgrounds in all of the features that had more than one related test or outline scenario.

### CODE REVIEW

File: cypress/integration/step-definitions/filters/the-user-adds-a-new-filter.when.js [hunk 1]
```diff
@@ -1,12 +1,14 @@
 import { When } from 'cypress-cucumber-preprocessor/steps';
-import { clickElement, fillField, elementIsVisible} from '../../utils/driver';
+import { clickElement, fillField, elementIsVisible, getElement} from '../../utils/driver';
 import { addFilterButton, filterSuggestionList, filterOperatorList, filterParams, saveFilterButton, selectedOperator} from '../../pageobjects/filters/filters.page';
 When('The user adds a new filter', () => {
   elementIsVisible(addFilterButton);
   clickElement(addFilterButton);
   fillField(filterSuggestionList,'rule.level');
+  getElement(filterOperatorList).should('not.be.disabled');
```
Comment (mauceballos) [inline cypress/integration/step-definitions/filters/the-user-adds-a-new-filter.when.js:8]: Should be a function on driver.js

File: cypress/integration/step-definitions/management/decoders/navigate-to-decoders.when.js [hunk 2]
```diff
@@ -13,5 +13,5 @@ When('The user navigates to decoders', () => {
   clickElement(managementButton);
   elementIsVisible(decodersLink);
   clickElement(decodersLink);
-  validateURLIncludes('/manager/?tab=decoders');
+  //validateURLIncludes('/manager/?tab=decoders');
```
Comment (mauceballos) [inline cypress/integration/step-definitions/management/decoders/navigate-to-decoders.when.js:16]: Should be delete this line, we shouldn't hade comment lines

**Review by mauceballos — APPROVED** [review]

File: cypress/integration/step-definitions/filters/the-user-adds-a-new-filter.when.js (hunk 1 above)
Comment (Mayons95) [reply cypress/integration/step-definitions/filters/the-user-adds-a-new-filter.when.js]: Done !

File: cypress/integration/step-definitions/management/decoders/navigate-to-decoders.when.js (hunk 2 above)
Comment (Mayons95) [reply cypress/integration/step-definitions/management/decoders/navigate-to-decoders.when.js]: Done

**Pull request merged**